survey/dashboards/config.py

```diff
@@ -0,0 +1,124 @@
+import os
+import json
+from django.conf import settings
+
+
+SORT_CONFIG_PATH = os.path.join(
```

Comment:
Config should probably be based on individual survey.survey_config rather than the JSON file?